docs/standard-library.rst

```diff
@@ -38,8 +38,15 @@ It behaves exactly like the generic `structlog.BoundLogger` except:
 
 - it's slightly faster due to less overhead,
 - has an explicit API that mirrors the log methods of standard library's `logging.Logger`,
+- it has correct type hints,
 - hence causing less cryptic error messages if you get method names wrong.
 
+----
+
+If you're using static types (e.g. with mypy) you also may want to use `structlog.stdlib.get_logger()` that has the appropriate type hints if you're using `structlog.stdlib.BoundLogger`.
+Please note though, that it will neither configure nor verify your configuration.
```

Comment:
The new docs section reads nicely and is not overwhelming or frightening, but I don’t know what to make of this line.  I have to decide if I want a runtime check or correct static typing?

Reply:
No, you never get the runtime check. I just don’t want people to expect any extra magic, just because the get_logger lives in the stdlib package.

Reply:
Ah!  The way it is written made me think that this was a specific downside to this new function, opposed to previous/regular behaviour.